include/fs/romfs.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+#include "helpers.hpp"
+#include <string>
+#include <vector>
+#include <memory>
+
+namespace RomFS {
+
+    struct RomFSNode {
+        std::u16string name {};
+        // The file/directory offset relative to the start of the RomFS
+        u64 offset { 0 };
+        u64 size { 0 };
+        bool isDirectory { false };
```

Comment:
I understand why this method of construction exists but 99% of the time it is not used for its intended purpose and it looks terrible